hs/server.hs

```diff
@@ -69,17 +79,38 @@ main = do
              dir graph $ graphResponse,
              dir code $ seeOther fbAuth1Url $ toResponse test,
              dir test $ look "code" >>= getEmail,
+             dir testPost $ look "code" >>= postToFacebook,
              dir about $ aboutResponse,
              dir static $ serveDirectory EnableBrowsing [] staticDir,
-             dir course $ path (\s -> liftIO $ queryCourse s)
+             dir course $ path (\s -> liftIO $ queryCourse s),
+             dir postFB $ seeOther fbAuth1UrlPost $ toResponse test
            ]
 
+postToFacebook :: String -> ServerPart Response
+postToFacebook code = (liftIO $ performPost code) >> graphResponse
+
+args2 :: FB.Argument
+args2 = ("message", "Test post please ignore")
+
+performPost :: String -> IO Response
+performPost code = withManager $ \manager -> FB.runFacebookT app manager $ do
```

Comment:
I should really stop working when I write something like that.

I believe I have found the solution.

I can't test it yet though, I can't get through to Facebook (even as a regular user). Will test tomorrow.

![fberrir](https://cloud.githubusercontent.com/assets/6301000/5914558/d373f2b0-a5c5-11e4-81ef-f6e0a9d5cac7.png)